support-data-rest/src/test/java/org/uniprot/api/support/data/statistics/controller/StatisticsControllerIT.java

```diff
@@ -129,6 +129,67 @@ void getByReleaseAndTypeAndMultipleCategories() throws Exception {
                 .andExpect(jsonPath("$.results[1].items[1].entryCount", is(9457)));
     }
 
+    @Test
+    void getAllByAttributeAndStatisticsType() throws Exception {
+        this.mockMvc
+                .perform(get("/statistics/history/fungi?statisticType=reviewed"))
+                .andDo(log())
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$.results.size()", is(1)))
+                .andExpect(jsonPath("$.results[0].statisticsType", is("REVIEWED")))
+                .andExpect(jsonPath("$.results[0].releaseName", is("2021_03")))
+                .andExpect(jsonPath("$.results[0].releaseDate", is("2021-05-24")))
+                .andExpect(jsonPath("$.results[0].valueCount", is(35360)))
+                .andExpect(jsonPath("$.results[0].entryCount", is(35360)));
+    }
+
+    @Test
+    void getAllByAttributeAndStatisticsType_emptyResults() throws Exception {
+        this.mockMvc
+                .perform(get("/statistics/history/insecta?statisticType=unreviewed"))
+                .andDo(log())
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$.results.size()", is(0)));
+    }
+
+    @Test
+    void getAllByAttributeAndStatisticsType_emptyResultsWithoutStatisticTypeSpecified()
```

Comment:
The method name says that the request is without statisticType, but we have it in the implementation. Can you please make it consistent?